src/de.rs

```diff
@@ -554,7 +556,7 @@ mod test {
 
         let expected = Empty;
         let expected_json = json!(null);
-        let code = "None";
+        let code = c"None";
```

Comment:
I think we can use `pyo3::ffi::c_str!` as an MSRV-compatible alternative, so that tests can run on MSRV.

Reply:
Thanks! Hadn't realised how new `CStr` literals were.